hashtag-counter/Controllers/HashTagController.cs

```diff
@@ -26,34 +25,39 @@ public class HashTagController : ControllerBase
 
         public HashTagController(ILogger<HashTagController> logger, IConfiguration config)
         {
+            Console.WriteLine("ctor.");
             this.logger = logger;
             this.configuration = config;
         }
 
         [HttpPost("messagebinding")]
         public async Task<IActionResult> PostMessageBinding([FromBody]SocialMediaMessage message)
         {
-            if (this.configuration[AppSettings.AppType] != AppSettings.HashTagCounter)
-            {
-                return BadRequest(new HTTPResponse($"Illegal request for {this.configuration[AppSettings.AppType]}"));
-            }
+            Console.WriteLine("enter messagebinding");
+
+            Console.WriteLine($"{message.CreationDate}, {message.CorrelationId}, {message.MessageId}, {message.Message}, {message.Sentiment}");
+            this.logger.LogTrace($"{message.CreationDate}, {message.CorrelationId}, {message.MessageId}, {message.Message}, {message.Sentiment}");
```

Comment:
logger doesn't work, does it ?

Reply:
Yes, I'm finding it doesn't log to console in k8s so it doesn't show up.